CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# 1.0.1 - 2016-12-08
+Updated Node and NPM versions
+
+# 1.0.0 - 2016-12-08
```

Comment:
:cow2: It would be great to follow the same Changelog version as we use in streambed: http://keepachangelog.com/en/0.3.0/